rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -561,8 +561,7 @@ def rebuild_bq_participant(p_id, ps_bqgen=None, pdr_bqgen=None):
 
     # filter test or ghost participants if production
     if app_id == 'all-of-us-rdr-prod':  # or app_id == 'localhost':
-        if ps_bqr.is_ghost_id == 1 or not ps_bqr.hpo or ps_bqr.hpo == 'TEST' or \
-            not ps_bqr.email or '@example.com' in ps_bqr.email:
+        if ps_bqr.is_ghost_id == 1 or ps_bqr.hpo == 'TEST':
```

Comment:
why the removal of `example` email ?

Reply:
If any show up in BigQuery they can be filtered out there.